proto/o5/dempe/v1/service.proto

```diff
@@ -44,37 +53,41 @@ message GetMessageRequest {
 
 message CapturedMessage {
   DeadMessage cause = 1;
-
 }
 
 message GetMessageResponse {
   CapturedMessage message = 1;
-  repeated MessageAction actions = 2;
+
+  message GetMessageAction {
+    MessageAction action = 1;
+    o5.auth.v1.Actor actor = 2;
+    google.protobuf.Timestamp timestamp = 3;
+  }
+  repeated GetMessageAction actions = 2;
 }
 
 message MessageAction {
+  string id = 1 [
+    (buf.validate.field).string.uuid = true
+  ];
 
   string note = 2;
 
-  o5.auth.v1.Actor actor = 3;
-
-  google.protobuf.Timestamp timestamp = 4;
```

Comment:
These have been moved to only exist as part of the `GetMessageResponse` as metadata about a singular action.